apps/api/src/test-data/buildTermTestData.ts

```diff
@@ -69,14 +191,19 @@ export const buildTermsForVocabularyList = (): Term[] =>
         // audioFilename: 'https://coscrad.org/wp-content/uploads/2023/06/503.mp3'
         // },
     ]
-        .map((partialDto) => ({
-            ...partialDto,
-            published: true,
-            text: buildBilingualText(partialDto.term, partialDto.termEnglish),
-            type: ResourceType.term,
-            isPromptTerm: false,
-        }))
-        .map((dto) => new Term(dto))
+        .filter((result): result is ResultOrError<Term> => !isNotFound(result))
+        .filter(validAggregateOrThrow)
+        .map(convertAggregatesIdToUuid);
+
+const buildStandAloneTerms = () =>
+    [
+        Term.fromEventHistory(bilingualTerm1Events.as({ id: '1' }), '1'),
```

Comment:
Do a map so you only have to wrap in the `Term.fromEventHistory` once.

Reply:
It's tricky because the ID is different in each case. We might want to use an array of tuples. Let's worry about finding a good pattern on the next one.